README.md

```diff
@@ -40,3 +40,19 @@ Contributions are very welcome, as are feature requests and suggestions. Please
 [codecov-url]: https://codecov.io/gh/JuliaData/CSV.jl
 
 [issues-url]: https://github.com/JuliaData/CSV.jl/issues
+
+## Alternatives
+
+There are several other packages for reading CSV files in Julia, which may suit your needs better:
+
+* The standard library contains [DelimitedFiles.jl](https://docs.julialang.org/en/v1/stdlib/DelimitedFiles/),
+  which is perfect for quickly reading small files.
+
+* [CSVFiles.jl](https://github.com/queryverse/CSVFiles.jl) uses the [FileIO.jl](https://github.com/JuliaIO/FileIO.jl) API 
+  into any [IterableTables.jl](https://github.com/queryverse/IterableTables.jl) sink.
+  The package uses [TextParse.jl](https://github.com/queryverse/TextParse.jl) for parsing.
```

Comment:
I copied this text from https://github.com/queryverse/TextParse.jl . But it's not terribly helpful for knowing which to choose. Both packages use Tables.jl, what's the most obvious sign you ought to use one or the other?

Reply:
Paging @davidanthoff -- how best should CSVFiles.jl be described?

Reply:
I think I wouldn't mention the IterableTables.jl story, because one should be able to materialize CSVFiles into any Tables.jl sink as well. The main difference here is really just that CSVFiles.jl uses a different parsing code than CSV.jl.

Reply:
I think it could be worth mentioned the FileIO.jl integration, since I think some people prefer the `load`/`save` api.

Reply:
Perhaps:
```suggestion
* [CSVFiles.jl](https://github.com/queryverse/CSVFiles.jl) and uses the [FileIO.jl](https://github.com/JuliaIO/FileIO.jl) API,
  but otherwise has similar goals. Like CSV.jl, it works with [Tables.jl](https://github.com/JuliaData/Tables.jl objects.
```